Work around threading issue on some AArch64 devices, see llvm/llvm-project#62165 for more info.
diff --git a/lld/ELF/Relocations.cpp b/lld/ELF/Relocations.cpp
index 2f7fcd664c6a..f9cf5e7c1a29 100644
--- a/lld/ELF/Relocations.cpp
+++ b/lld/ELF/Relocations.cpp
@@ -1181,7 +1181,7 @@
   std::atomic<size_t> next{0};
   // MIPS modifies MipsGotSection during relocation scanning, which is not
   // suitable for parallelism.
-  size_t numWorkers = ctx.arg.emachine == EM_MIPS
+  size_t numWorkers = ctx.arg.emachine == EM_MIPS || ctx.arg.emachine == EM_AARCH64
                           ? 1
                           : std::min<size_t>(ctx.arg.threadCount, numFiles + 1);
   parallelFor(0, numWorkers, [&](unsigned shard) {
